packages/react-devtools-shared/src/backend/console.js

```diff
@@ -30,6 +30,33 @@ export function isStringComponentStack(text: string): boolean {
   return PREFIX_REGEX.test(text) || ROW_COLUMN_NUMBER_REGEX.test(text);
 }
 
+const STYLE_DIRECTIVE_REGEX = /^%c/;
+
+function isStrictModeOverride(args: Array<string>, method: string): boolean {
```

Comment:
This function could also use a block comment above it explaining its purpose.

I _think_ it exists to differentiate between strict mode styling (e.g. `"%cSome message", "color: ..."`) and component stacks (e.g. `"%sSome message", "  at ..."`) but to someone less familiar with the history of this file, it might be confusing.